app/controllers/IncomeController.scala

```diff
@@ -19,7 +19,7 @@ package controllers
 import com.google.inject.name.Named
 import javax.inject.Inject
 import controllers.actions.ValidatePerson
-import controllers.auth.{AuthAction, AuthedUser}
+import controllers.auth.{AuthAction, AuthedUser, AuthenticatedRequest}
```

Comment:
Look like this is not used, please delete it

Reply:
removed AuthenticatedRequest import